@here/harp-mapview/lib/MapView.ts

```diff
@@ -2227,18 +2282,26 @@ export class MapView extends THREE.EventDispatcher {
     private setupCamera() {
         const { width, height } = this.getCanvasClientSize();
 
-        this.m_scene.add(this.m_camera); // ensure the camera is added to the scene.
+        const defaultGeoCenter = new GeoCoordinates(52.518611, 13.376111, 3000);
+
+        if (this.projection.type === ProjectionType.Spherical) {
```

Comment:
Here, there: 

> // ### TODO: RTC for spherical projection

Is RTC so complex that we can't deliver it right now?
Does lack of RTC introduce same precision issues ? (i assume yes)

In the other hand, for me this condition - here and there should read `if (!this.cameraRelativeToWorldCenter)` because this is more about RTC vs no RTC than spherical vs non-spherical.

Reply:
RTC is not difficult implement but there are couple of ways of implementing it. Currently I'm considering to implement it by using an additional internal/private THREE.Camera. I just didn't want to make this change too tricky to review. But I have no problem in waiting to merge until RTC is implemented.